public/app/pipeline-orchestrator.js

```
@@ -10,6 +10,83 @@
+// ─── AudioWorklet loader with CDN fallback ─────────────────────────────────
+// Primary path is same-origin /app/dsp-processor.js (COOP+COEP friendly).
+// If that fails (404, offline blob, etc.) we walk the manifest mirrors —
+// Vercel Blob → Cloudflare R2 → Hugging Face Hub — fetch the source text,
+// wrap it in a same-origin Blob URL, and pass that to addModule(). The Blob
+// URL is same-origin so it satisfies COEP require-corp without needing the
+// remote response to carry CORP headers.
+const _WORKLET_FALLBACK_CACHE = { manifest: null, blobUrl: null };
+async function _getWorkletManifest() {
+  if (_WORKLET_FALLBACK_CACHE.manifest) return _WORKLET_FALLBACK_CACHE.manifest;
+  try {
+    const resp = await fetch('/app/models-manifest.json', { credentials: 'omit' });
```

ESLint

[error] 'fetch' is not defined.

(no-undef)